sympy/solvers/tests/test_ode.py

```diff
@@ -2577,7 +2576,7 @@ def test_almost_linear():
 
     eq = x*f(x)*d + 2*x*f(x)**2 + 1
     sol = dsolve(eq, f(x), hint = 'almost_linear')
-    assert sol[0].rhs == -sqrt(C1 - 2*Ei(4*x))*exp(-2*x)
+    assert sol[0].rhs == -sqrt((C1 - 2*Ei(4*x))*exp(-4*x))
```

Comment:
Why has this changed?

Reply:
Can you change this test so it checks both solutions?
```python
sol = {
    Eq(f(x), -sqrt((C1 - 2*Ei(4*x))*exp(-4*x))),
    Eq(f(x), sqrt((C1 - 2*Ei(4*x))*exp(-4*x)))
}
assert set(dsolve(eq), f(x), hint=our_hint) == sol
```

Reply:
okay